content/docs/practices/naming.md

```diff
@@ -26,9 +26,9 @@ A metric name...
     (exported by many client libraries)
   * <code><b>http</b>\_request\_duration\_seconds</code>
     (for all HTTP requests)
-* ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
-* ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers).See below for a list of base units.
-* ...should have a suffix describing the unit, in plural form. Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable. Also note that this applies to units in the narrow sense (like the units in the table below), but not to countable things in general. For example, <code>connections</code> or <code>notifications</code> are not considered units for this rule and do not have to be at the end of the metric name. (See also examples in the next paragraph.)
+* ...MUST have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
+* ...SHOULD use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See [below](#base-units) for a list of base units.
+* ...SHOULD have a suffix describing the unit, in plural form. Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable. Also note that this applies to units in the narrow sense (like the units in the table below), but not to countable things in general. For example, <code>connections</code> or <code>notifications</code> are not considered units for this rule and do not have to be at the end of the metric name. (See also examples in the next paragraph.)
```

Comment:
I find the "units ... countable" part hard to follow.  As far as I can see there are no examples with `connections` or `notifications`, so referring to examples is perverse.  
Is this emphasis because we don't put `requests` in the UNIT metadata?

Given the long section you have added below, maybe remove this part and refer to that section?

Reply:
These words have been added as a reaction to misunderstandings in the past. We can move it elsewhere, but it should stay prominent enough to avoid repeating the misunderstanding.

There _is_ an example below: 

```
* <code>http\_requests\_<b>total</b></code>
    (for a unit-less accumulating count)
```

Reply:
"narrow sense units" (: 

While odd it makes sense for names and it's something we recommend for years. I will keep this form for now, we can change later.